### ISSUE TITLE
Classic Version does not compile on clean CocoaPods install

### ISSUE THREAD

**Issue by AndrewLipscomb** [alexdrone/Render#102]
Tested with a new Single Page App from XCode (9.2) with a Cocoapods install file 

```
target 'TestRender' do
        use_frameworks!
	pod 'Render'
end
```

The lockfile indicates I have Render 4.9.1

The application does not compile - the following errors are given when compiling the first time

TestRender/Pods/Render/Render/lists/CollectionNode.swift:41:70: Contextual closure type '(UICollectionView, YGLayout, CGSize) -> (Void)' expects 3 arguments, but 1 was used in closure body

TestRender/Pods/Render/Render/lists/CollectionNode.swift:59:70: Contextual closure type '(UICollectionView, YGLayout, CGSize) -> (Void)' expects 3 arguments, but 1 was used in closure body

TestRender/Pods/Render/Render/lists/CollectionNode.swift:108:62: Contextual closure type '(NilState, CGSize) -> NodeType' expects 2 arguments, but 1 was used in closure body

TestRender/Pods/Render/Render/core/ComponentView.swift:169:83: Contextual closure type '(TimeInterval, AnyComponentView, CGSize) -> ()' (aka '(Double, AnyComponentView, CGSize) -> ()') expects 3 arguments, but 1 was used in closure body

TestRender/Pods/Render/Render/core/ComponentView.swift:309:31: Cannot call value of non-function type 'String?'

TestRender/Pods/Render/Render/core/ComponentView.swift:332:57: Cannot call value of non-function type 'String?'

TestRender/Pods/Render/Render/core/ComponentViewCell.swift:56:34: Cannot assign value of type '(_) -> ()' to type '(TimeInterval, AnyComponentView, CGSize) -> ()' (aka '(Double, AnyComponentView, CGSize) -> ()')

TestRender/Pods/Render/Render/core/ComponentViewCell.swift:114:66: Cannot call value of non-function type 'String?'

Surprisingly, I get additional errors when I press compile a second time

TestRender/Pods/Render/Render/helpers/Console.swift:127:24: 'dynamic' instance method 'willEnterForeground()' must also be '@objc'

TestRender/Pods/Render/Render/helpers/Console.swift:131:24: 'dynamic' instance method 'didEnterBackground()' must also be '@objc'

TestRender/Pods/Render/Render/helpers/Console.swift:157:24: 'dynamic' instance method 'timerDidFire(timer:)' must also be '@objc'

**Comment by AndrewLipscomb** [comment]
Also, I happened to run the install again cleanly with the verbose options

```
-> Installing Render (4.9.1)
 > Git download
 > Git download
     $ /usr/local/bin/git clone https://github.com/alexdrone/Render.git
     /var/folders/xf/29dsn5rj5gq1415vxkc33kk40000gn/T/d20180308-947-1l6a2i5
     --template= --single-branch --depth 1 --branch 4.9.1
     Cloning into '/var/folders/xf/29dsn5rj5gq1415vxkc33kk40000gn/T/d20180308-947-1l6a2i5'...
     Note: checking out '2a4c7365d9b9208c248725523c8b4544db3238d4'.
     
     You are in 'detached HEAD' state. You can look around, make experimental
     changes and commit them, and you can discard any commits you make in this
     state without impacting any branches by performing another checkout.
     
     If you want to create a new branch to retain commits you create, you may
     do so (now or later) by using -b with the checkout command again. Example:
     
       git checkout -b <new-branch-name>
     
     Checking out files: 100% (315/315), done.
  > Copying Render from
  `/Users/swarmfarm/Library/Caches/CocoaPods/Pods/Release/Render/4.9.1-02c15` to
  `Pods/Render`
```
It seems the CocoaPods install is not at the same commit as the current commit d953ad2. I can't say I'm good with this yet (coming from a C++/Web React background) so I'm not sure how CocoaPods organises its packaging, and if that behaviour is normal. 

Edit: Checked the .framework's last commit hash - its that checked out one. I'm guessing that's an intentional checkout in CocoaPods.

**Comment by AndrewLipscomb** [comment]
Just found the other closed issues for the 4.9.1 issue. I can close this....but maybe its better to get that CocoaPod up to date..?
